src/views/ForgotPassword.vue

```diff
@@ -19,25 +19,38 @@
                 :placeholder="$t('general.placeholder.email')"
             />
 
-            <button
+            <router-link class="font-medium text-indigo-600 hover:text-indigo-500 text-sm" to="/login">
+              {{ $t('app.routerTitle.login') }}
+            </router-link>
+
+            <base-button
                 type="submit"
                 class="flex justify-center w-full px-4 py-2 my-4 text-sm font-medium text-white bg-indigo-600 border border-transparent rounded-md shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
                 :disabled="!valid"
             >
-              {{ $t('general.resetPassword.title') }}
-            </button>
+              {{ $t('general.submit') }}
+            </base-button>
           </form>
         </ValidationObserver>
+        <img
+          v-if="loading"
+          class="h-5 w-5 mx-auto my-3"
+          src="https://icon-library.com/images/loading-icon-transparent-background/loading-icon-transparent-background-12.jpg"
+        >
```

Comment:
I should remove this after the BaseButton version of @LAlex14 will be merged (the one with the loading image inside the button)